### DISCUSSION TITLE
Coral Mini PCIe adapter

### DISCUSSION THREAD

**Discussion by racingpassion** [blakeblackshear/frigate#2458] [Ask A Question]
Does anyone know of an adapter that works with the Mini PCIe coral unit, either to USB or PCIe.

**Comment by T89L** [comment]
Ableconn PEXM2150E PCI Express x1 Adapter Card with M.2 Key E Socket - Support M2 E Key or A-E Key WiFi or Bluetooth Module

For this one specifically - https://coral.ai/products/m2-accelerator-dual-edgetpu/

You will only get one of the TPUs though. Check the dual tpu discussion for more information on getting both TPUs. You will need a different card but it won't be available for some time.

**Reply by southwestflavorz** [reply]
The question is not about the M.2 cards. It's about converting the Mini PCIe to a desktop PCIe.

**Comment by paultanger** [comment]
This one worked for me: Ableconn PEX-MP117 Mini PCI-E to PCI-E Adapter Card https://www.amazon.com/dp/B07JBCL1CJ?psc=1&ref=ppx_yo2ov_dt_b_product_details

**Reply by southwestflavorz** [reply]
When you say worked for you, do you mean with ESXi, or native Linux?

**Reply by paultanger** [reply]
It was plug and play, I have a supermicro board, and run truenas scale (so debian based) and run within kubernetes.

**Reply by jamesawebb** [reply]
I'm also after an adapter for the [(half-size) mini PCIe](https://coral.ai/products/pcie-accelerator/) - but to USB.  Any pointers appreciated.
